src/utils/cancelRequests.ts

```diff
@@ -39,19 +41,24 @@ export const getAxiosReqParams = (
     cache: defaultCache,
   };
 
-  if (!reqConfig) {
+  const reqConfigWithDefault = {
+    ...getDefaultRequestsConfig(),
+    ...reqConfig,
+  };
+
+  if (!reqConfigWithDefault) {
     return axiosReqConfig;
   }
 
-  if (reqConfig.cancelToken) {
-    axiosReqConfig.cancelToken = reqConfig.cancelToken.getToken();
+  if (reqConfigWithDefault.cancelToken) {
+    axiosReqConfig.cancelToken = reqConfigWithDefault.cancelToken.getToken();
   }
-  axiosReqConfig.retries = reqConfig.retries;
-  if (reqConfig.cache) {
-    axiosReqConfig.cache = reqConfig.cache;
+  axiosReqConfig.retries = reqConfigWithDefault.retries;
```

Comment:
I think you could also check if property `retries` is set/defined on `reqConfigWithDefault` object to avoid setting it as undefined  (same a as above),

Reply:
True, thank you, fixed! 

(I haven't really changed this code, only renamed the variable - but it makes sense to fix this)